static/js/components/ErrorMessage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Alert from 'react-bootstrap/lib/Alert';
+
+
+export default class ErrorMessage extends React.Component {
+  static propTypes = {
+    errorInfo:   React.PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { errorInfo: {error_code: errorCode, user_message: userMessage} } = this.props;
+
+    let errorCodeStr, userMessageStr = '';
```

Comment:
FYI only `userMessageStr` is being set to a string here, the other is `undefined`. Both will be treated the same way by React AFAIK but we should choose one or the other here